CMakeLists.txt

```diff
@@ -71,9 +71,20 @@ add_compile_definitions(
 )
 
 find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGLWidgets Svg OpenGL LinguistTools Core5Compat Xml Quick REQUIRED)
-
-set(REQUIRED_FF7TK_VERSION 0.83.0.0)
```

Comment:
Please keep the version requirement info . 
it makes sure the  Api is compatible with makou

Reply:
We have that hardcoded on the `vcpkg.json` manifest from now on, and it's easier to manage ( centralize the dependency management instead of having to change this string on multiple files )

Reply:
Not everyone will be using vcpkg so please leave this for the people who are not.

Reply:
Done